src/MarginTrading.AssetService/Workflow/Products/StartProductsSaga.cs

```diff
@@ -32,7 +32,7 @@ public class StartProductsSaga
         [UsedImplicitly]
         public async Task Handle(EodProcessFinishedEvent e, ICommandSender sender)
         {
-            var productsResult = await _productsRepository.GetAllAsync(null, null, isStarted: false);
+            var productsResult = await _productsRepository.GetAllAsync(null, null, null, isStarted: false);
```

Comment:
Same, makes sense to use named arguments for better readability.

```suggestion
            var productsResult = await _productsRepository.GetAllAsync(mdsCodeFilter: null, mdsCodes: null, productIds: null, isStarted: false);
```